pre-docs/writing_plans.md

```diff
@@ -189,7 +189,7 @@ fail_plan($errorobject)
 
 ### Catching Errors in a Plan
 
-Bolt includes a `catch_errors` function which executes a block of code and returns the error if an error is raised or the result of the block if no errors are raised. When you call `run_plan` with `_catch_errors`, use a `catch_errors` block, or call the `error` method on a result, you may get an error.
+Bolt includes a `catch_errors` function which executes a block of code and returns the error if an error is raised or the result of the block if no errors are raised. You may get an `Error` object returned if you call a function with `_catch_errors`, use a `catch_errors` block, or call the `Error` function.
```

Comment:
Not quite sure what "use a `cat_errors` block, or call the `Error` function", previously the instruction was "when you do x, use catch error", not it read "here is some info, use catch error".

Reply:
Well, the other 2 concepts are introduced earlier in the doc. So this introduces the 3rd concept, then says "If you use any of these concepts you'll get an Error object. Here's what an Error object is"

Reply:
ohhhh, I see, you are enumerating the cases you would expect an error object to be returned. That makes sense.